.github/workflows/release-cli.yml

```diff
@@ -29,11 +48,19 @@ jobs:
 
       - uses: Swatinem/rust-cache@v1.2.0
 
-      - name: Build ${{ matrix.target }} release binary
-        run: cargo build --target=${{ matrix.target }} --release --package swap --bin swap
+      - name: Install compiler for armhf arch
+        if: matrix.target == 'armv7-unknown-linux-gnueabihf'
+        run: |
+          sudo apt-get update
+          sudo apt-get install gcc-arm-linux-gnueabihf
+
+      - name: Build ${{ matrix.target }} ${{ matrix.bin }} release binary
+        env:
+          CARGO_TARGET_ARMV7_UNKNOWN_LINUX_GNUEABIHF_LINKER: arm-linux-gnueabihf-gcc
+        run: cargo build --target=${{ matrix.target }} --release --package swap --bin ${{ matrix.bin }}
 
       - name: Smoke test the binary
-        run: target/${{ matrix.target }}/release/swap --help
+        run: target/${{ matrix.target }}/release/${{ matrix.bin }} --help
```

Comment:
I thought you didn't want to smoke test the arm one?

Reply:
Need an `if` that skips this step on the `armv7-unknown-linux-gnueabihf` target then.

Reply:
I wonder how that slipped out... it only works with the if, thanks for spotting that one, should be fixed now!